llvm/lib/Transforms/PeepholeOptimizationCourse/ObfuscateI8AddFunction/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_llvm_component_library(
+	LLVMObfuscateI8AddFunction
```

Comment:
I don't see where you link against that library. How did linker find implentations?

Reply:
fix it here [f93f764](https://github.com/ggladilov/llvm-project/pull/6/commits/f93f7642514533f5c74f78c3e2af6e082cccd8e4)